emscripten.py

```diff
@@ -273,50 +271,48 @@ def move_preasm(m):
   function_table_sigs = list(function_table_data.keys())
 
   in_table, debug_tables, function_tables_defs = make_function_tables_defs(
-    implemented_functions, all_implemented, function_table_data, settings, metadata)
+    implemented_functions, all_implemented, function_table_data, metadata)
 
   exported_implemented_functions = get_exported_implemented_functions(
-    all_exported_functions, all_implemented, metadata, settings)
+    all_exported_functions, all_implemented, metadata)
 
-  asm_setup = create_asm_setup(debug_tables, function_table_data, metadata, settings)
-  basic_funcs = create_basic_funcs(function_table_sigs, settings)
-  basic_vars = create_basic_vars(exported_implemented_functions, forwarded_json, metadata, settings)
+  asm_setup = create_asm_setup(debug_tables, function_table_data, metadata)
+  basic_funcs = create_basic_funcs(function_table_sigs)
+  basic_vars = create_basic_vars(exported_implemented_functions, forwarded_json, metadata)
 
-  shared.Settings.copy(settings)
```

Comment:
This line always assumed we were in the same process anyway.

Reply:
I suspected that since we never tested the out of process case that it no longer worked, nice to see confirmation.